Ported/ariab/JobCloth/Assets/Scripts/BufferCloth/ClothProjectSystem.cs

```diff
@@ -1,23 +1,22 @@
 ﻿using Unity.Burst;
-using Unity.Collections;
 using Unity.Entities;
 using Unity.Jobs;
 using Unity.Mathematics;
-using UnityEngine;
 
 [UpdateInGroup(typeof(SimulationSystemGroup))]
 [UpdateAfter(typeof(ClothTimestepSystem))]
 public class ClothProjectSystem : JobComponentSystem
 {
     [BurstCompile]
-    struct ProjectClothPositionsJob : IJobForEach_BBBBC<ClothProjectedPosition, ClothCurrentPosition, ClothPreviousPosition, ClothPinWeight, ClothTimestepData>
+    struct ProjectClothPositionsJob : IJobForEach_BBBBCC<ClothProjectedPosition, ClothCurrentPosition, ClothPreviousPosition, ClothPinWeight, ClothTimestepData, ClothWorldToLocal>

```

Comment:
Out of curiosity, why is it necessary to use the explicit `IJobForEach_*` form here instead of plain ol' `IJobForEach`?

Reply:
Buffers! Plain ol' `IJobForEach` doesn't work with buffers :(

Reply:
Buffeeeeeeeers! *shakes fist*